Task_Position_the_Cubes/index.html

```diff
@@ -1,15 +1,22 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Cubes Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding from the lines below for POSITION THE CUBES TASK; DO NOT TOUCH the code above-->
-  </body>
-</html>
```

Comment:
Consider adding content inside the child div elements to make the cubes visible on the webpage.